### PR TITLE
Update versions in PA for Release 24.10

### PR DESCRIPTION
**Pull request by pvijayakrish**
Updates the README file for the 24.10 Triton release.

### CODE REVIEW

File: templates/template_vars.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 General:
-  release: 24.09
+  release: 24.10
   triton_cli_version: 0.0.11
-  genai_perf_version: 0.0.7dev
+  genai_perf_version: 0.0.7
```
Comment (mc-nv) [inline templates/template_vars.yaml:4]: Aren't version should be:
```suggestion
  genai_perf_version: 0.0.8dev
```

Comment (fpetrini15) [reply templates/template_vars.yaml:4]: This is being merged into a release branch, so I believe `0.0.7` is the correct version.

**Review by fpetrini15 — APPROVED** [review]

File: templates/template_vars.yaml (hunk 1 above)
Comment (mc-nv) [reply templates/template_vars.yaml:4]: @ganeshku1 , could you please advise which version of "genai-perf" will be used for 24.11 release?

Comment (pvijayakrish) [reply templates/template_vars.yaml:4]: @mc-nv **0.0.7** is the correct version to be used for genai-perf. Please refer https://github.com/triton-inference-server/perf_analyzer/blob/r24.10/genai-perf/genai_perf/__init__.py

**Pull request merged**